test_regress/t/t_cover_else_points.pl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2003-2009 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+compile(
+    verilator_flags2 => [ "--binary   --coverage  --timing -Wall" ],
+    );
+
+execute(
+    all_run_flags => [" +verilator+coverage+file+$Self->{obj_dir}/coverage.dat"],
+    check_finished => 1,
+    );
+
+# Read the input .v file and do any CHECK_COVER requests
+inline_checks();
+
+run(cmd => ["../bin/verilator_coverage",
+            "--annotate-points",
+            "--annotate", "$Self->{obj_dir}/annotated",
+            "$Self->{obj_dir}/coverage.dat",
+            ],
+    verilator_run => 1,
+    );
```

Comment:
```suggestion
    ) if !$self->errors && !$self->skips;
```

As Ubuntu 20 is failing due to a skip from needing coroutines.

Reply:
That does not seem to work locally. I am getting the following error. 
```
- t/t_cover_else_points.v:59: Verilog $finish
Can't call method "errors" on an undefined value at t/t_cover_else_points.pl line 25.
Compilation failed in require at ./driver.pl line 842.
```
Line 25 is the beginning for the `run` command.

Reply:
My fault, should be Self, capitalized.